scm-ui/ui-webapp/src/repos/components/form/RepositoryForm.tsx

```diff
@@ -237,6 +257,9 @@ class RepositoryForm extends React.Component<Props, State> {
               onChange={this.toggleInitCheckbox}
               helpText={t("help.initializeRepository")}
             />
+            {this.state.initRepository && (
+              <ExtensionPoint name="repos.create.initialize" props={extensionProps} renderAll={false} />
```

Comment:
Do we only have one repository initializer? It is not possible to define multiple?